### ISSUE TITLE
swig error : Unrecognized option -builtin

### ISSUE THREAD

**Issue by saikat8018** [CRPropa/CRPropa3#157]
While installing CRPropa3 on Unix system, the following error occurs. Can anyone please tell me, what does this means and how to remove this error?

```
[ 87%] Generating crpropa_wrap.cxx
swig error : Unrecognized option -builtin
Use 'swig -help' for available options.
make[2]: *** [crpropa_wrap.cxx] Error 1
make[1]: *** [CMakeFiles/crpropa-swig.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by TobiasWinchen** [comment]
I guess that your swig version is outdated, what version are you using ($swig -version)?
You can either use a recent version of swig or try disabling builtin feature by settingthe according  cmake option (cmake .. -DENABLE_SWIG_BUILTIN=OFF)

**Comment by saikat8018** [comment]
I am using SWIG version 4.0.0. Is that a problem?

**Comment by adundovi** [comment]
SWIG 4.0.0 is still not officially released, and the last released version is 3.0.12 according to their [github](https://github.com/swig/swig/releases). I am afraid that we cannot support unreleased/development versions of 3rd party libraries and programs.

**Comment by saikat8018** [comment]
Oh! Okay. Thank you.
